### DISCUSSION TITLE
Previous Movie Not Obtaining theme

### DISCUSSION THREAD

**Discussion by SiriosDev** [LizardByte/community#5] [Themerr]
### Describe the Bug

movies uploaded before themerr was added to my plex server, do not receive theme songs

### Expected Behavior

all movies (present in thmerrDB) receive the theme in my plex server

### Additional Context

I tried `update metadata` and `analyze`

**Comment by ReenigneArcher** [comment]
Is this a new agent or legacy agent library?

If legacy share a screenshot of the agent priority from Plex settings.

**Comment by SiriosDev** [comment]
Nope, I use Plex for my Movie Library, but I will post the screenshot anyway

![image](https://github.com/LizardByte/Themerr-plex/assets/26876994/cdfa9076-31c0-445c-8f59-06edba1cdf25)

**Comment by ReenigneArcher** [comment]
That's not what I asked or the screenshot I need. Which agent is your library using? You can get the agent from the library settings.

![image](https://github.com/LizardByte/Themerr-plex/assets/42013603/f71c1d5f-7fb4-46e6-9f28-1e024d99e589)

**Comment by SiriosDev** [comment]
As mentioned before I use Plex Movie

![image](https://github.com/LizardByte/Themerr-plex/assets/26876994/0248dc29-bf99-4617-8c79-1ede84fa5e08)

**Comment by ReenigneArcher** [comment]
I misread the issue. In any event, this is a support question. Please use https://app.lizardbyte.dev/discord for support.
